client/components/organization-wrapper/organization-wrapper.js

```diff
@@ -303,7 +303,17 @@ export default class OrganizationWrapper extends React.Component {
           ) : null}
           {cssPath && orgSlug ? (
             <Helmet>
-              <link rel="stylesheet" href={getAssetPath(orgSlug, cssPath)} />
```

Comment:
I suggest doing some pre-processing in the beginning of this JS function above, eg:

```javascript
if (!css) css = [];  // not sure this is needed, just ensure we always have a CSS array.
if (cssPath) css.push(cssPath);
```

This way the block of code below be simplified to always expect only an array and nothing else.